index.html

```diff
@@ -1,20 +1,450 @@
+          </p>
+          <p>
+            The <a href="#dfn-user-agent">user agent</a> MUST let the end-user choose which
+            <a data-cite="screen-capture#dfn-display-surface">display surfaces</a> to share
+            out of all available choices every time, and MUST NOT use
+            any {{MediaTrackConstraints}} in <var>options</var>.<code>video</code>
+            or <var>options</var>.<code>audio</code> to limit that choice.
+          </p>
+          <p>
+            The <a href="#dfn-user-agent">user agent</a> MUST return a list of {{MediaStream}}
+            objects, where each stream belongs to exactly one of the
+            <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>
+            selected by the user. In case not all the
+            <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>
+            selected by the user can be captured, {{MediaDevices/getDisplayMediaSet}} must return
+            a promise <a>rejected</a> with a new {{InvalidStateError}}.
+          </p>
+          <!--
+              This paragraph is intended to avoid duplicate (and potentially out-of-sync) paragraphs
+              with the getDisplayMedia spec. Is this too imprecise?
+            -->
+          <p>
+            The <a href="#dfn-user-agent">user agent</a> must apply the {{DisplayMediaStreamOptions.audio}}
+            and {{DisplayMediaStreamOptions.video}} constraints to to each of the selected
+            <a data-cite="screen-capture#dfn-display-surface">display surfaces</a> in the same way as it does for
+            {{MediaDevices/getDisplayMedia()}}.
+          </p>
+          <p>
+            The <a href="#dfn-user-agent">user agent</a> MAY allow the user to select a mix of different
+            <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>.
+          </p>
+          <!-- TODO: Is this really necessary? -->
+          <p>
+            If {{DisplayMediaStreamOptions.systemAudio}} is set to {{SystemAudioPreferenceEnum/"include"}},
+            the <a href="#dfn-user-agent">user agent</a> MAY add an audio track capturing system audio
+            for each returned {{MediaStream}} object or MAY restrict the audio surce further,
+            depending on the <a data-cite="screen-capture#dfn-display-surface">display surfaces</a> selected by the
+            user.
+          </p>
+          <!-- TODO: Should we allow this behavior with gDMS? -->
+          <p>
+            The <a href="#dfn-user-agent">user agent</a> MUST return a promise <a>rejected</a> with
+            a new {{TypeError}} if the {{DisplayMediaStreamOptions.surfaceSwitching}}
+            is specified.
+          </p>
+          <p>
+            As with {{MediaDevices/getDisplayMedia()}}, {{PermissionState/"granted"}} permissions
+            cannot be persisted.
+          </p>
+
+
+          <p>When the {{MediaDevices/getDisplayMedia()}}
```

Comment:
The more I'm reading this paragraph, the more I feel like this will go out of sync with `getDisplayMedia()`. Shall we create hooks in `getDisplayMedia` spec so that `getDisplayMediaSet` could reuse them?